ui/components/app/transaction-icon/transaction-icon.scss

```diff
@@ -1,8 +1,8 @@
 .transaction-icon {
   &__grey-circle {
-    height: 28px;
-    width: 28px;
-    border-radius: 14px;
+    height: 32px;
+    width: 32px;
+    border-radius: 16px;
     background: var(--color-background-alternative);
```

Comment:
Let's use [AvatarIcon](https://metamask.github.io/metamask-storybook/?path=/docs/components-componentlibrary-avataricon--docs) here. We won't have to write extra css then